Common/HttpWebRequestWrapperLib/HttpWebRequestWrapper.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Text;
+
+namespace HttpWebRequestWrapperLib
+{
+    /// <summary>
+    /// Wrapper of http web requests and responses that encapsulate GET, PUT POST and DELETE requests
+    /// </summary>
+    public class HttpWebRequestWrapper
+    {
+        public HttpWebRequest httpWebRequest { get; set; }
+        public HttpWebResponse httpWebResponse { get; set; }
+
+        public HttpWebRequestWrapper(string contentType = "application/json")
+        {
+            httpWebRequest.ContentType = contentType;
+        }
+        
+        public string Get(string url, Dictionary<string,string> parameters)
```

Comment:
I'm not sure, but it seems to me sometimes we need a body in get request.

Reply:
No, we don't